tools/static_mode_white_list.py

```diff
@@ -13,702 +13,289 @@
+    'test_batch_fc_op', 'test_c_comm_init_all_op', 'test_conv2d_fusion_op',
+    'test_dataset_dataloader', 'test_fleet_metric', 'test_fused_bn_add_act',
+    'test_fused_multihead_matmul_op', 'test_ir_inplace_pass',
+    'test_mix_precision_all_reduce_fuse', 'test_parallel_executor_pg',
+    'test_rank_attention_op', 'test_fleet_base', 'test_fleet_graph_executor',
+    'test_fleet_meta_optimizer_base', 'test_ir_memory_optimize_transformer',
+    'test_trt_fc_fuse_pass', 'test_trt_quant_conv2d_dequant_fuse_pass',
+    'test_trt_slice_plugin', 'test_trt_transpose_flatten_concat_fuse_pass',
+    'test_mean_op', 'test_build_strategy_fusion_group_pass',
+    'test_coalesce_tensor_op', 'test_dataset', 'test_fleet_base_single',
+    'test_fleet_rolemaker_new', 'test_fused_fc_elementwise_layernorm_op',
+    'test_fusion_transpose_flatten_concat_op', 'test_ir_memory_optimize_nlp',
+    'test_nvprof', 'test_pipeline', 'test_weight_decay', 'test_fleet_base_2',
+    'test_fleet_pipeline_meta_optimizer', 'test_fleet_checkpoint',
+    'test_ir_fusion_group_pass', 'test_trt_pad_op',
+    'test_trt_shuffle_channel_detect_pass', 'test_trt_subgraph_pass',
     'test_parallel_executor_seresnext_base_gpu',
     'test_parallel_executor_seresnext_with_fuse_all_reduce_gpu',
-    'test_parallel_executor_seresnext_with_reduce_gpu',
-    'test_sync_batch_norm_op',
+    'test_parallel_executor_seresnext_with_reduce_gpu', 'test_sync_batch_norm_op',
     'test_multiprocess_dataloader_iterable_dataset_static',
-    'test_multiprocess_dataloader_static',
-    'test_load_op_xpu',
-    'test_activation_op_xpu',
-    'test_adam_op_xpu',
-    'test_assign_op_xpu',
-    'test_batch_norm_op_xpu',
-    'test_cast_op_xpu',
-    'test_concat_op_xpu',
-    'test_elementwise_add_op_xpu',
-    'test_fill_constant_op_xpu',
-    'test_gather_op_xpu',
-    'test_matmul_op_xpu',
-    'test_matmul_v2_op_xpu',
-    'test_mean_op_xpu',
-    'test_momentum_op_xpu',
-    'test_reduce_mean_op_xpu',
-    'test_reduce_sum_op_xpu',
-    'test_reshape2_op_xpu',
-    'test_sgd_op_xpu',
-    'test_shape_op_xpu',
-    'test_slice_op_xpu',
-    'test_generate_proposals_v2_op',
-    'test_lamb_op_xpu',
-    'test_model_cast_to_bf16',
-    'test_sgd_op_bf16',
-    'test_marker_op',
+    'test_multiprocess_dataloader_static', 'test_load_op_xpu',
+    'test_activation_op_xpu', 'test_adam_op_xpu', 'test_assign_op_xpu',
+    'test_batch_norm_op_xpu', 'test_cast_op_xpu', 'test_concat_op_xpu',
+    'test_elementwise_add_op_xpu', 'test_fill_constant_op_xpu',
+    'test_gather_op_xpu', 'test_matmul_op_xpu', 'test_matmul_v2_op_xpu',
+    'test_mean_op_xpu', 'test_momentum_op_xpu', 'test_reduce_mean_op_xpu',
+    'test_reduce_sum_op_xpu', 'test_reshape2_op_xpu', 'test_sgd_op_xpu',
+    'test_shape_op_xpu', 'test_slice_op_xpu', 'test_generate_proposals_v2_op',
+    'test_lamb_op_xpu', 'test_model_cast_to_bf16', 'test_sgd_op_bf16',
+    'test_marker_op', 'test_trunc_op'
```

Comment:
add ',' after each line, even the last line.
![image](https://user-images.githubusercontent.com/6888866/120753699-28e9fe80-c53e-11eb-93cd-d562c62c6ac0.png)